### ISSUE TITLE
Comparison with `axios-jwt` library

### ISSUE THREAD

**Issue by plush-mesh** [mvanroon/react-native-axios-jwt#25]
Hello, thanks for the library! Looks like you took inspiration from `axios-jwt` (https://github.com/jetbridge/axios-jwt) library. I want to ask you how do they both compare, I mean there was a need some time back, but what's the purpose of `react-native-axios-jwt` now when `axios-jwt` supports React Native from Sep 20, 2022?

**Comment by mvanroon** [comment]
Hi @plush-mesh, I was actually the one that added react-native support to axios-jwt back in september.

Since then, some changes have been added to axios-jwt that haven't been properly tested in all different environments (web, native and server) causing some issues to pop up. Since I use this package myself in quite some projects myself, I've decided to maintain this repo for the time being.

TLDR: If axios-jwt works for your project, use that. If not, use this one. They're very similar.

**Comment by plush-mesh** [comment]
Thank you for clarification!

**Comment by antgel** [comment]
Interesting thread. I have just tried adding `axios-jwt` to my Expo project and in the simplest use cases I get strange errors like:

```
 WARN  Possible Unhandled Promise Rejection (id: 0):                                  
TypeError: Cannot read property 'getItem' of undefined                                
```

in `getRefreshToken` and `getAuthTokens`. I haven't successfully debugged yet, but I think somehow `axios-jwt` isn't using (defining) `AsyncStorage` as it is meant to.

I'll try this library out instead, and I'd be happy to hear what the "issues" are with `axios-jwt` that you're referring to. Thanks!

**Comment by mvanroon** [comment]
Can’t remember exactly, sorry 😰
